app/assets/stylesheets/adopt.css.scss

```diff
@@ -35,3 +36,58 @@
     font-weight: normal;
   }
 }
+
+.list-of-rats {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  
+  li {
+    display: inline-block;
+    height: 150px;
+    margin: 100px 1em 1em 0;
+    position: relative;
+    width: 150px;
+    
+    &:hover span.text-content {
```

Comment:
To satisfy the hound, you can remove the `span` keyword. For `Selector should have depth of applicability no greater than 2, but was 3` I think this may be fine to ignore.